Makefile.vofiles

```diff
@@ -28,6 +28,14 @@ VFILES := $(THEORIESV) $(CONTRIBV)
 # Build directory is needed to have the glob files
 BUILD_VFILES=$(addprefix $(VO_OUT_DIR),$(VFILES))
 
+BUILD_VFILES_FILE=$(BUILD_OUT_DIR).all_v_files
+
+# This has to be kept in sync with above, used in the windows build to
+# avoid coqdep maximum number arguments problems. Paths in file have
+# to be relative to the location of the coqdep input file.
+$(BUILD_VFILES_FILE): $(BUILD_VFILES)
```

Comment:
The dep is on VFILES not BUILD_VFILES

Reply:
You are right, I was thinking of running coqdep on the copied sources as they should include auto-generated .v files if we ever had them; let me think about this one.